spark-connector/spark-connector/build.gradle.kts

```diff
@@ -40,3 +41,26 @@ dependencies {
   testImplementation("org.scala-lang.modules:scala-java8-compat_$scalaVersion:$scalaJava8CompatVersion")
   testRuntimeOnly(libs.junit.jupiter.engine)
 }
+
+sourceSets {
+  main {
+    scala {
+      setSrcDirs(listOf("src/main/scala", "src/main/java"))
```

Comment:
The `GravitinoDriverPlugin` references the scala class, so need to the compile scala classes  before java classes